week-2/D-predicates/exercise.js

```diff
@@ -7,7 +7,7 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+  return typeof number <= 0
```

Comment:
Did you test this one properly? Think about what return values you'd expect from `isNegative()` and what values `typeof` returns.

Reply:
Hello Jonny, 

Thank you for your kind comments. You have given me a lot to work with.

I found this section did not click with me and made a note to look up some further tutorials on the topic. 

It was not meant to upload it and will try to be more careful in future!

Reply:
@mtbdata711 this one might help: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators

Reply:
Thanks!

Reply:
I've uploaded a revision of this. Let me know if it was what you meant.